lib/pgfinder/matching.py

```diff
@@ -574,6 +569,7 @@ def consolidate_results(
     def pick_from_highest_intensity_instance(column: pd.Series):
         return column[df.loc[column.index, intensity_column].idxmax()]
 
+    print(f"{df.columns=}")
```

Comment:
Is this an intentional addition?

Reply:
Gah, I must remember to search the Magit buffer of changes for prints when committing in the future, thanks for spotting this.